server-common/src/main/java/org/apache/kafka/server/common/TestFeatureVersion.java

```diff
@@ -32,6 +32,8 @@ public enum TestFeatureVersion implements FeatureVersion {
 
     public static final String FEATURE_NAME = "test.feature.version";
 
+    public static final TestFeatureVersion LATEST_PRODUCTION = TEST_1;
```

Comment:
I wonder if this can cause bugs if latestTesting = latestProduction

Reply:
Makes sense. Added a check to the value